cmd/worker/main.go

```diff
@@ -35,7 +36,28 @@ const (
 	localPkgPathFmt = "/local/%s"
 )
 
-func handleMessage(ctx context.Context, msg *pubsub.Message, packagesBucket *blob.Bucket, resultsBucket, fileWritesBucket, imageTag string) error {
+type pkgIdentifier resultstore.PkgIdentifier
```

Comment:
nit: no need for this. We can just use the `resultstore.PkgIdentifier` directly.